### DISCUSSION TITLE
RFC location in project

### DISCUSSION THREAD

**Discussion by CalvinRodo** [cds-snc/docs#43] [General]
The ideas for RFCs is eventually to provide cross-organization documentation of how we work, we are restricting it to dev for now because we haven't had a chance to discuss this with other organizations. 

I recommend we move RFCs out of the `dev` folder and move them back to an RFC folder to the base of the folder and add a community section so we can identify which communities they apply to.

**Comment by dj2** [comment]
The community tags is a great idea.

I think before we move RFCs back to the top level we should discuss with the various communities and make sure they'd be on board with using RFCs. If a few of them are, then we should move back up and tag. If dev is the only community on board I think this location makes sense.

**Comment by wmoussa-gc** [comment]
The discussion with the other communities have started and we're looking at presenting the concept and what we've done so far in the next show the thing. As for where to put the rfcs folder, I lean more in keeping it the way it is right now. Eventually every community will have its own folder, all sitting under the same repo. If with time we feel we should merge the all, we could. 

As for the community metatag, I think it is a a good idea too. Do you want to update the template? I think we should also agree on a list of tags..https://github.com/cds-snc/docs/blob/main/development/rfc-template.md

**Reply by CalvinRodo** [reply]
Sounds good, if we are planning on merging the RFCs might make more sense to move to a base folder when that is agreed upon, might be a bit of work to merge them if other teams start doing RFCS and we have clashing numbers.
